browser/admin/src/integrator/WordBook.ts

```diff
@@ -366,7 +364,7 @@ class WordBook {
 		buttonContainer.appendChild(cancelButton);
 
 		const submitButton = document.createElement('button');
-		submitButton.textContent = 'Submit';
+		submitButton.textContent = 'Save';
```

Comment:
how will we handle l18n?

Reply:
As of now, we aren't. I think we need to accept language from Nextcloud first? 

Maybe we will handle it in upcoming PRs.